src/system/tests/TestSystemObject.cpp

```diff
@@ -80,15 +80,15 @@ class TestObject : public Object
 private:
     enum
     {
-        kPoolSize = 122 // a multiple of kNumThreads, less than CHIP_SYS_STATS_COUNT_MAX
+        kPoolSize = 112 // a multiple of kNumThreads, less than CHIP_SYS_STATS_COUNT_MAX
     };
     static ObjectPool<TestObject, kPoolSize> sPool;
```

Comment:
How about we add a static_assert that `kPoolSize % kNumThreads == 0`?  And perhaps that `kPoolSize < CHIP_SYS_STATS_COUNT_MAX`.

Reply:
Acked